LabGui.py

```diff
@@ -288,7 +296,7 @@ def __init__(self, argv=[]):
             )
             logging.warning(msg)
             # default setting
-            Tool.INTF_GPIB = Tool.INTF_PROLOGIX
+            Tool.INTF_GPIB = Tool.INTF_VISA # DEFAULT SHOULD BE PYVISA, MUCH MORE COMPATIBLE
```

Comment:
but also proprietary...

Moreover, a GPIB-usb converter by National Instrument costs 800 CAD, a prologix one 150 CAD. I didn't experienced troubles with the prologix converter. Maybe it depends on the instruments?

Reply:
Although I do agree with the proprietary part, all the drivers were originally designed to work with PyVISA. I think it would be a fantastic idea to include compatibility with as many alternate GPIB-USB converters as possible, since NI converters are very expensive, but it is also most widely used in labs as far as I am aware. Moreover, if we set INTF_GPIB by default to be Prologix, that means unless changed manually by the user, it will only work with Prologix. There are other GPIB-USB converters (for example, ones made by Agilent) who's drivers either install their own visa backend, or utilize the ni backend, for compatibility with visa. Thats why I believe that by default the GPIB interface should be set to PyVISA.

Also, by beginning to redesign the PrologixController class to be a prologix replacement of the visa.Resource.GPIBInstrument class, I believe full compatibility can be attained, especially if we model any future GPIB interfaces in this manner